civihr_employee_portal/features/leave_and_absences_feature/leave_and_absences_feature.info

```diff
@@ -0,0 +1,13 @@
+name = Leave and Absences Feature
+description = Contains all the sections related to Leave and Absences section.
+core = 7.x
+package = Compucorp
+dependencies[] = block
+dependencies[] = civihr_employee_portal
+dependencies[] = civihr_leave_absences
+dependencies[] = civicrm_resources
+dependencies[] = ctools
```

Comment:
@AkA84 we are using pages to create My Leave page, so thats has dependencies for ctools.